app/src/main/java/com/kwartracker/android/transactions/ui/add/AddTransactionFragment.kt

```diff
@@ -0,0 +1,41 @@
+package com.kwartracker.android.transactions.ui.add
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.kwartracker.android.R
+import com.kwartracker.android.databinding.FragmentTransactionAddBinding
+import com.kwartracker.android.transactions.ui.list.ListTransactionsFragment
+import com.kwartracker.android.transactions.ui.main.TransactionsFragment
+
+class AddTransactionFragment : Fragment() {
```

Comment:
@danacealfonso 
Provide more comprehensible class name. What is the difference of this fragment to `AddNewTransactionFragment`?

Reply:
fix in 8e6fa36a048fe726e08f61764cfae1f06a381b94